cmp/grid/Grid.ts

```diff
@@ -205,10 +205,10 @@ export class GridLocalModel extends HoistModel {
             defaultColDef: {
                 sortable: true,
                 resizable: true,
-                suppressHeaderContextMenu: true,
```

Comment:
This doesnt appear to be a deprecated property? Did you find some reporting that this was no longer working?
https://www.ag-grid.com/javascript-data-grid/column-properties/#reference-header-suppressHeaderContextMenu